import_models_to_verta/mlflow_databricks_bulk_import_example.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
Could I suggest putting this in `registry/` rather than creating a new top-level directory?

Reply:
Done